CHANGELOG.md

```diff
@@ -1,26 +1,63 @@
 # Changelog
 
-## Unreleased
+## icu4x 2.0-beta2
 
 - Components
     - General
         - Some crates have been given the ability to be built without `alloc` (unicode-org#6077, unicode-org#6078)
+        - Consistently wrap all options bags in None in Rust and FFI (unicode-org#6084)
+        - Remove `Vn` from data structs, `Marker` from data markers (unicode-org#6070)
+        - Data markers have been renamed to be consistent (unicode-org#4991)
+        - Options now consistently live in `options` modules (unicode-org#6138)
     - `icu_provider`
         - There is now a `data_marker!` macro (unicode-org#6072)
+        - Remove `AnyProvider` (unicode-org #6086)
     - `icu_locale`
         - `LocaleExpander`, `LocaleDirectionality`, and `LocaleCanonicalizer` distinguish between `new_common()` and `new_extended()` constructors (unicode-org#5958)
     - `icu_segmenter`
         - Segmenters that can take a content locale now specify `_root()` on their default localeless constructors (unicode-org#5958)
     - `icu_calendar`
         - Rename IsoWeekday to Weekday (unicode-org#6140)
+        - Add `Date::wrap_calendar_in_ref` (unicode-org#6016)
+        - `Time` and `DateTime` moved to `icu_time` component (unicode-org#5961)
+        - Audit exhaustiveness of icu_calendar::types (unicode-org#5978)
+        - Add calendar argument to IXDTF parsing (unicode-org#5982)
+        - Use correct day of year for Japanese (unicode-org#5992)
+    - `icu_casemap`
+        - Add Borrowed variants to casemapper types (unicode-org#6088)
+    - `icu_casemap`
+        - Add missing keys to `CollatorPreferences` (unicode-org#5950)
+    - `icu_collections`
+        - Remove some panics from zerovec's derive and icu_codepointtrie (unicode-org#6052)
     - `icu_properties`
         - Add to/from ICU4C APIs to enumerated properties (unicode-org#6091)
         - `GeneralCategoryGroup::contains` now accepts `self` by value (unicode-org#5952)
+    - `icu_time`
+        - Renamed from `icu_timezone`
+        - `Time` and `DateTime` moved to `icu_time` component (unicode-org#5961)
+        - Dumb down DateTime (unicode-org#5985)
+
 - Utils
-    - `zerovec`
+    - `env_preferences`: `0.1.0 -> 0.2.0`
+        - Fix timezone naming convention (unicode-org#5954)
```

Comment:
this crate is not published, it was incorrectly bumped to 0.1.0 in https://github.com/unicode-org/icu4x/pull/5857, and should still be at 0.0.0

Reply:
can you downgrade the version in the crate? i think with the cargo msrv unpublished crates might not even need a version